### PR TITLE
User can login

### PR DESCRIPTION
**Pull request by julieahlquist**
## PT-Link https://www.pivotaltracker.com/story/show/167284712

User Story: 
_As a user,
In order to post to the forum and buy/sell plants,
I would like to be able to login to my account on Florum_

Installing devise in order to enable user login

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
```
Comment (oliverochman) [inline app/views/devise/registrations/new.html.erb]: if you do not change anything here I want you to delete it

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```
Comment (oliverochman) [inline app/views/devise/registrations/edit.html.erb]: out of scope

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (erikwesterberg) [reply app/views/devise/registrations/new.html.erb]: done

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (erikwesterberg) [reply app/views/devise/registrations/edit.html.erb]: done

Comment (oliverochman) [reply app/views/devise/registrations/edit.html.erb]: you should delete this file

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (oliverochman) [reply app/views/devise/registrations/new.html.erb]: you should delete this file

**Review by oliverochman — APPROVED** [review]

**Pull request merged**